receive_test.go

```diff
@@ -178,7 +240,7 @@ func TestCopySimple(t *testing.T) {
 		"ADD zzz/bb dir",
 		"ADD zzz/bb/cc dir",
 		"ADD zzz/bb/cc/dd symlink ../../",
-		"ADD zzz.aa zzdata",
+		"ADD zzz.aa file zzdata",
```

Comment:
This was confusing - was the omission of the file type intended? Let me know if this is the case and how to interpret the test then. I had to add the file type as I updated the changes language parser to panic on unexpected input.